vxquery-core/src/main/java/org/apache/vxquery/datamodel/values/ValueTag.java

```diff
@@ -74,6 +74,8 @@
     public static final int XS_NMTOKENS_TAG = BuiltinTypeConstants.XS_NMTOKENS_TYPE_ID;
     public static final int XS_ENTITIES_TAG = BuiltinTypeConstants.XS_ENTITIES_TYPE_ID;
 
+    public static final int JS_NULL_TAG = BuiltinTypeConstants.JS_NULL_ID;
```

Comment:
Can you also add js:null constant to XDMConstants.

        JS_NULL_CONSTANT = new byte[1];
        JS_NULL_CONSTANT[0] = ValueTag.JS_NULL_TAG;

Along with the setJsNull(IPointable p) method